docs/testing/ior.md

```diff
@@ -83,10 +101,10 @@ Finished            : Thu Apr 29 23:26:03 2021
 
 ## Run mdtest
 
-Use mdtest to create 30K files:
+Use mdtest to create 30K files using `API=POSIX` (as for IOR, using `-a DFS` will provide much better performance):
 
 ```sh
-$ mpirun -hostfile /path/to/hostfile_clients -np 10 mdtest -a POSIX -z 0 -F -C -i 1 -n 3334 -e 4096 -d /tmp/daos_dfuse/ -w 4096
+$ mpirun -hostfile /path/to/hostfile_clients -np 10 <your_dir>/bin/mdtest -a POSIX -z 0 -F -C -i 1 -n 3334 -e 4096 -d /tmp/daos_dfuse/ -w 4096
```

Comment:
Still -a Posix.  Please update for better performance. 

Please help new to mdtest users understand what -a -z -F -C -i -n -e -D -W all mean.